templates/default/kafka-restore.sh.erb

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash 
+  
+<%= node['mysql']['bin_dir'] %>/mysql -u root \
+        --skip-password \
+        -S <%= node['ndb']['mysql_socket'] %> \
+        -B -N -e "select topic_name, num_partitions, num_replicas from hopsworks.project_topics"  | sed 's/\t/,/g' > /tmp/topics.csv
+
+while IFS="," read -r topic_name num_partitions num_replicas 
```

Comment:
It's better if we guarded against old values without `num_partitions` and `num_replicas`. Just skip restoring them, print them once the script has finished and return with an abnormal exit code. I think putting some default values such as 1 and silently ignore it might create issues.

Reply:
Ok, in general I tested it and it works, it the `kafka-topic.sh` prints that it can't understand what NULL is, but I guess I can check it before hand. Let me fix it